rmgpy/rmg/input.py

```diff
@@ -693,6 +693,18 @@ def uncertainty(localAnalysis=False, globalAnalysis=False, uncorrelated=True, co
     }
 
 
+def restart_from_seed(path=None, relative_path=True):
```

Comment:
Could you make this an argument in the other options block instead of it's own block?

I don't think you need to ask about relative_path. You can check whether the path is absolute using `os.path.isabs`, and if not, assume (and say in the documentation) that it is relative to the input file.

Reply:
I updated the absolute path issue, thanks for the suggestion!

As for moving this to the other options block I feel that this is not really an option but rather a feature that completely alters the behavior of a normal RMG job. I don't want this to be hidden away in the input file where the user might forget to change this. It is also easier to automatically write restart files if this is its own block, though this can of course be overcome.

I am willing to discuss this further, and change it if you feel strongly about it, but I do feel like it is different from the other options in the block